src/components/Metrics.js

```diff
@@ -128,7 +128,11 @@ function TokenPrice({ token }) {
 
   return (
     <div>
-      <Metric label="Honey price" value={`$${price}`} color={theme.green} />
+      <Metric
+        label="Honey price"
+        value={`$${formatDecimals(price, 2)}`}
```

Comment:
What's the reason for adding this? Note that we are already fixing it to 2 decimal positions https://github.com/1Hive/honey-pot/blob/8d9c08c9d60ddbe05f1feaf9fdf992ae7d8af65b/src/hooks/useUniswapHNYPrice.js#L39

Reply:
To keep the a coherent format with the rest of values in dollars. The converted prices use the `formatDecimals` function, that puts comma separators. I realized that when the price of HNY is >$1,000, it doesn't have that comma separator.

Reply:
Ah good catch!